src/structs/Api.ts

```diff
@@ -22,10 +22,12 @@ interface APIOptions {
 /**
  * Top.gg API Client for Posting stats or Fetching data
  * @example
+ * ```js
  * const Topgg = require(`@top-gg/sdk`)
  *
  * const api = new Topgg.Api('Your top.gg token')
- * @link https://topggjs.rtfd.io <- Library docs
+ * ```
+ * @link https://topgg.js.org <- Library docs
  * @link https://docs.top.gg <- API Reference
```

Comment:
Shouldn't this be in the format of
```suggestion
 * Library docs: {@link https://topgg.js.org}
 * API Reference: {@link https://docs.top.gg}
```
https://tsdoc.org/pages/tags/link/

Reply:
This would work even better
```js
 /**
 * {@link https://topgg.js.org | Library docs}
 * {@link https://docs.top.gg | API Reference}
 */
```

Reply:
Since it doesn't actually set a JSDoc property with this, it just turns into this
![image](https://user-images.githubusercontent.com/7953076/124167981-88bac300-da72-11eb-8673-54fe09e9eb11.png)
I honestly prefer the separate link tags, though I'm not sure.

Reply:
```ts
/**
 * @link {@link https://topgg.js.org | Library docs}
 * @link {@link https://docs.top.gg | API Reference}
 */
 ```
 seems to do exactly what we need
![image](https://user-images.githubusercontent.com/7953076/124171812-05e83700-da77-11eb-911d-55836e1e47a7.png)
is that good?